build-tools/src/main/kotlin/org/jetbrains/kotlin/benchmark/BenchmarkingPlugin.kt

```diff
@@ -158,6 +159,8 @@ abstract class BenchmarkingPlugin: Plugin<Project> {
             project.afterEvaluate {
                 linkerOpts.addAll(project.benchmark.linkerOpts)
             }
+
+            freeCompilerArgs = project.benchmark.compilerOpts + project.compilerArgs
```

Comment:
It's better to move this into `afterEvaluate` too because user may change `project.benchmark.compilerOpts`.